### DISCUSSION TITLE
skip webauth login page and redirect to OIDC provider login page?

### DISCUSSION THREAD

**Discussion by amigthea** [open-webui/open-webui#8167] [Q&A]
I'm trying to find a way to disable the login page, go directly on OIDC provider login page and avoid click on "Continue with <provider name>", is that possible?

**Comment by AaronZ345** [comment]
This is supported now. With your existing OIDC settings, add:

```env
ENABLE_OAUTH_SIGNUP=true
ENABLE_LOGIN_FORM=false
OAUTH_AUTO_REDIRECT=true
```

Then restart Open WebUI.

Automatic redirect only activates when exactly one OAuth provider is configured and neither LDAP nor trusted-header authentication is enabled. If more than one provider is available, Open WebUI keeps the provider-selection page.

Keep `/auth?form=true` as an emergency route while testing the change. Also make sure `WEBUI_URL` matches the public origin and that the provider's redirect URI points to:

```text
https://<your-open-webui-host>/oauth/oidc/callback
```

If persistent configuration is enabled, an older value stored through the Admin UI can take precedence over a changed environment value, so verify the effective OAuth settings there as well.
